service/internal/graph/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+type: profile
```

Comment:
This is type does not make a lot of sense to me, but I am not sure which one to suggest

Reply:
this is probably a copy/paste error. i'll try to find something else to put there

Reply:
i adapted this to class: service, type: graph
maybe that works